src/Intercom/Clients/UsersClient.cs

```diff
@@ -335,21 +335,43 @@ public User IncrementUserSession(String id)
             return result.Result;
         }
 
+        public User IncrementUserSession(String id, List<String> companyIds)
```

Comment:
Before we only allowed you to increment a user session passing an ID but that wouldn't increment any company sessions as we need a POST with both the user and company details. This allows that.